decisions/mnm_hermetics_decisions.txt

```diff
@@ -348,75 +349,98 @@ targeted_decisions = {
 		ai_check_interval = 12
 
 		potential = {
+
 			# Warcraft
 			is_untouchable_trigger = no
-			
-			is_member_of_hermetics_trigger = yes
-			society_rank = 4
-			prisoner = no
-			NOT = { is_incapable = yes }
+			society_member_of = twilights_hammer_cult
+			religion = old_gods_worship
+			true_religion = old_gods_worship
+			society_rank >= 3
+
 		}
 
 		allow = {
-			FROM = {
-				custom_tooltip = {
-					text = hermetics_wrote_magnum_opus_cd_tooltip
-					NOT = { has_character_flag = wrote_magnum_opus }
+
+			# Warcraft
+			prisoner = no
+			NOT = { is_incapable = yes }
+			custom_tooltip = {
+				text = twilights_hammer_cult_wrote_whispers_cd_tooltip
+				NOT = { has_character_modifier = wrote_whispers_cooldown_modifier }
+			}
+			custom_tooltip = {
+				text = whispers_tooltip_tip
+			}
```

Comment:
This tooltip is not about requirements. So, what its doing in requirements?

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/75231682-ec6e1580-57c6-11ea-900b-94e6839449a0.png)

</details>

Reply:
moved to effect